networkx/algorithms/connectivity/connectivity.py

```diff
@@ -199,15 +199,12 @@ def local_node_connectivity(
         raise nx.NetworkXError("Invalid auxiliary digraph.")
 
     kwargs = {"flow_func": flow_func, "residual": residual}
-    if flow_func is shortest_augmenting_path:
-        kwargs["cutoff"] = cutoff
-        kwargs["two_phase"] = True
-    elif flow_func is edmonds_karp:
-        kwargs["cutoff"] = cutoff
-    elif flow_func is dinitz:
-        kwargs["cutoff"] = cutoff
-    elif flow_func is boykov_kolmogorov:
-        kwargs["cutoff"] = cutoff
+    match flow_func.__name__:
+        case "shortest_augmenting_path":
+            kwargs["cutoff"] = cutoff
+            kwargs["two_phase"] = True
+        case "edmonds_karp" | "dinitz" | "boykov_kolmogorov":
+            kwargs["cutoff"] = cutoff
```

Comment:
This is better than the existing code, but even better would be:
```python
kwargs["cutoff"] = cutoff
if flow_func is shortest_augmenting_path:
    kwargs["two_phase"] = True
```

Reply:
I thought of this solution, and it is indeed elegant. But I guessed that the flow functions are not  exhaustively listed, like maybe there is some flow function that should not be matched to edit its kwargs.